README.md

```diff
@@ -92,6 +91,35 @@ tail = true
 click-right = spotifatius toggle-liked
 ```
 
+## config.toml options
+
+### format
+
+The JSON output can be formatted using the `format` var in config.toml:
```

Comment:
```suggestion
🗒 **Note:** These formatting options are not yet final and might change in upcoming versions. They won't be considered breaking changes as a result, though a minor version bump will be done if this changes. [More info.](https://github.com/AndreasBackx/spotifatius/pull/8)

The JSON output can be formatted using the `format` var in config.toml:
```

Let's just say that this might change a bit still. I'm happy with your contributions, though I don't want to have to deal with people being annoyed in the future when it might change. Though it'll be done with care regardless.